src/Mod/Arch/ArchWall.py

```diff
@@ -1167,7 +1167,14 @@ def attach(self,vobj):
         self.Object = vobj.Object
         from pivy import coin
         tex = coin.SoTexture2()
-        tex.image = Draft.loadTexture(Draft.svgpatterns()['simple'][1], 128)
+
+        ## TODO - https://github.com/FreeCAD/FreeCAD/pull/2746 Backmerge Werner's fix
+        #tex.image = Draft.loadTexture(Draft.svgpatterns()['simple'][1], 128)
+        image = Draft.loadTexture(Draft.svgpatterns()['simple'][1], 128)
+        if not image is None:
+            tex.image = image
+        ## TODO
```

Comment:
What is the purpose of leaving a comment with this information? Comments and links to previous commits or forum threads should only be done if there is an important explanation in them. This change does not need any further explanation. Also, don't use TODO as if it were parentheses.

Reply:
Done further cleaning